src/options/quantifiers_options.toml

```diff
@@ -201,13 +201,37 @@ name   = "Quantifiers"
   help       = "use model-based quantifier instantiation"
 
 [[option]]
-  name       = "mbqiFastSygus"
+  name       = "mbqiEnum"
   category   = "expert"
-  long       = "mbqi-fast-sygus"
+  long       = "mbqi-enum"
   type       = "bool"
   default    = "false"
   help       = "use fast enumeration to augment instantiations from MBQI"
 
+[[option]]
+  name       = "mbqiEnumExtVarsGrammar"
+  category   = "expert"
+  long       = "ext-vars-grammar"
```

Comment:
This should be `mbqi-enum-ext-vars-grammar`, similarly the other 2 options need this prefix.